bin/init.bash

```diff
@@ -30,7 +30,11 @@ if [[ ${CK8S_CLOUD_PROVIDER} != "" ]]; then
 fi
 
 # Validate the flavor
-if [ "${flavor}" != "default" ] && [ "${flavor}" != "gcp" ] && [ "${flavor}" != "aws" ]; then
+if [ "${flavor}" != "default" ] && \
+   [ "${flavor}" != "gcp" ] && \
+   [ "${flavor}" != "citycloud" ] && \
```

Comment:
Not sure about this, but let me know what you think. What about skipping citycloud here and just treating it as generic openstack? If we want something specifically for citycloud we can add it later.

Reply:
Yes I agree. I mostly wanted to get this here to show my thinking. But since it seems to be the same for safespring (for what we know now) I think we can change it to openstack

Reply:
You certainly have a point there